### PR TITLE
Add support for showing / returing multiple locations

### PR DESCRIPTION
**Pull request by mnederlof**

### CODE REVIEW

**Comment by mnederlof** [comment]
We are currently working on having a rbd backend and a swift backend. In order to do so, we needed these changes.

File: tests/pillar/single_multiple_backends.sls [hunk 1]
```diff
@@ -0,0 +1,70 @@
+glance:
+  server:
+    enabled: true
+    version: liberty
```
Comment (pupapaik) [inline tests/pillar/single_multiple_backends.sls]: same here. Please put there version at least mitaka to see validity of config.

**Comment by pupapaik** [comment]
Please resolve small adjustment. THanks

**Review by pupapaik — CHANGES_REQUESTED** [review]

**Comment by mnederlof** [comment]
Updated the pillar test data

**Pull request merged**